test-suite/build.gradle

```diff
@@ -29,4 +30,15 @@ dependencies {
     testImplementation(projects.micronautMicrostreamCache)
     testImplementation(projects.micronautMicrostreamRest)
     testImplementation(libs.jupiter.jupiter.params)
+
+    testImplementation(libs.managed.microstream.sql)
+    testImplementation(mnSql.micronaut.jdbc.hikari)
+    testRuntimeOnly(mnSql.postgresql)
+}
+
+micronaut {
+    testResources {
```

Comment:
I think we should disable platform bom with `importMicronautPlatform = false`. see: https://github.com/micronaut-projects/micronaut-sql/pull/891/files

Reply:
Done in https://github.com/micronaut-projects/micronaut-microstream/pull/190/commits/80db913866041bc103e75e1b98a470d9550a7443